test/test_monitoring.py

```diff
@@ -239,6 +239,7 @@ def _test_find_options(self, query, expected_cmd):
             # Exhaust the cursor to avoid kill cursors.
             tuple(cursor)
 
+    @client_context.require_version_min(4, 4, 4)
```

Comment:
Why is this needed?

Reply:
"Comment" is not supported on "getMore" commands below 4.4.4 IIUC

Reply:
Doesn't that make this PR a major breaking change?

Reply:
I checked the [getMore docs](https://docs.mongodb.com/manual/reference/command/getMore/) and it appears that comment is not supported below 4.4.4. So yes, this would be a major breaking change.